### ISSUE TITLE
support AdGuard scriptlet syntax

### ISSUE THREAD

**Issue by antonok-edm** [brave/adblock-rust#324]
AdGuard uses `#%#//scriptlet(...)` for scriptlet injection syntax, whereas uBO filter lists generally use `+js(...)`. AdGuard's syntax requires quoting, which is supported in adblock-rust as of v0.8.3, so it could now be supported as an additional syntax alias.

**Comment by superlex** [comment]
Good idea. Currently AG filters lists use uBO scriptlet syntax `##+js(...)`, but since uBO now supports AG syntax `#%#//scriptlet(...)` probably in the future they will use their own syntax.

However, as of now I think is more important adblock-rust supports AdGuard cosmetic filter variants
(https://github.com/brave/adblock-rust/issues/236) and other procedural filters (https://github.com/brave/adblock-rust/issues/293) because this will improve quality adblocking a lot.
